server/errorstore/errors_test.go

```diff
@@ -399,18 +417,19 @@ func TestHttpHandler(t *testing.T) {
 		eh.ServeHTTP(res, req)
 
 		require.Equal(t, res.Code, 200)
-		var errs []struct {
-			Cause struct {
-				Message string
-			}
-			Wrap []struct {
-				Message string
-			}
-		}
 		require.NoError(t, json.Unmarshal(res.Body.Bytes(), &errs))
 		require.Len(t, errs, 2)
-		require.NotEmpty(t, errs[0].Cause.Message)
-		require.NotEmpty(t, errs[1].Cause.Message)
+		require.NotEmpty(t, errs[0].Error.Cause.Message)
+		require.NotEmpty(t, errs[1].Error.Cause.Message)
+
+		//order is not guaranteed when retrieving from redis
```

Comment:
nitpick: instead of using if else, should we sort the errors before asserting eg using `sort.Slice`?

Reply:
great catch, I was following a pattern used in the same file, but that's way more clear.